pkg/subctl/cmd/cloud/cloud.go

```diff
@@ -41,3 +49,10 @@ func init() {
 	restConfigProducer.AddKubeContextFlag(cloudCmd)
 	cmd.AddToRootCommand(cloudCmd)
 }
+
+// ExpectFlag exits with an error if the flag value is empty.
+func ExpectFlag(flag, value string) {
+	if value == "" {
+		exit.WithMessage(fmt.Sprintf("You must specify the %v flag", flag))
+	}
+}
```

Comment:
This function is already defined [here](https://github.com/submariner-io/submariner-operator/blob/0c4ef6c9942718116c8ca46983e03e079c000dbb/pkg/subctl/cmd/utils/utils.go#L63).